canonical/ubuntu.com
# WD-6314 - rebrand /ai/what-is-kubeflow

## Done

Update /ai/what-is-kubeflow design according to figma and copy doc
Need confirmation from Andreea 

## QA

- [demo link](https://ubuntu-com-13711.demos.haus/ai/what-is-kubeflow)
- [copy doc](https://docs.google.com/document/d/1m-96RlGkbzFa1KaGAxYFpY0ztIlsTuViCjXSAlErQlg/edit)
- [figma](https://www.figma.com/file/sGeCjFvA1kV9Kmvg2S2VBn/23.10-U.com---AI-bubble?node-id=248%3A1446&mode=dev)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to the demo and copy doc

## Issue / Card
[WD-6314](https://warthogs.atlassian.net/browse/WD-6314)

[WD-6314]: https://warthogs.atlassian.net/browse/WD-6314?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
